tests/component/test_export_signal_properties.py

```diff
@@ -0,0 +1,63 @@
+"""Contains a collection of pytest tests that validates export signal properties."""
+
+import pytest
+
+from nidaqmx.constants import ExportAction
+from nidaqmx.error_codes import DAQmxErrors
+from nidaqmx.errors import DaqError
+from nidaqmx.task import Task
+
+
+def test__export_signal__get_enum_property__returns_value(ai_voltage_chan_task: Task):
```

Comment:
Rename to ..._returns_default_value. Same for the other tests that validate default values.

Reply:
Renamed get and reset test cases to _returns_default_value